provider/aws_test.go

```diff
@@ -713,6 +722,210 @@ func TestAWSApplyDryRun(t *testing.T) {
 	}
 }
 
+func TestAWSCreateRecordsCNAME(t *testing.T) {
+	provider := newAWSProvider(t, false)
+
+	_, err := provider.CreateZone("ext-dns-test.teapot.zalan.do.")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	records := []endpoint.Endpoint{{DNSName: "create-test.ext-dns-test.teapot.zalan.do.", Target: "foo.elb.amazonaws.com"}}
+
+	err = provider.CreateRecords("ext-dns-test.teapot.zalan.do.", records)
```

Comment:
what about the scenario when record already exists ? (should check that error is returned)